IPCheck/parseAll.sh

```diff
@@ -40,14 +40,18 @@ DST_REPO=$2
 ./dashAnalysis.sh "Maven" 	"$SRC_REPO/smartclide-cicd"								"$DST_REPO/smartclide-cicd"
 ./dashAnalysis.sh "Docker"	"$SRC_REPO/smartclide-cicd-gitlab"						"$DST_REPO/smartclide-cicd-gitlab"
 ./dashAnalysis.sh "Maven" 	"$SRC_REPO/smartclide-context"							"$DST_REPO/smartclide-context"
-./dashAnalysis.sh "Other" 	"$SRC_REPO/smartclide-deployment-extension"				"$DST_REPO/smartclide-deployment-extension"
+./dashAnalysis.sh "Maven" 	"$SRC_REPO/smartclide-context/context-core"				"$DST_REPO/smartclide-context-core"
```

Comment:
Aren't these sub-modules of `smartclide-context` already covered by running dashAnalysis on the parent in line 42?